src/components/marquee-wizard/marquee-wizard.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="marquee-wizard-side-panel">
+
+    <div class="marquee-wizard-description-container" *ngIf="description">
+        <!-- If a template was provided display it -->
+        <ng-container *ngIf="isTemplate" [ngTemplateOutlet]="description"></ng-container>
+
+        <!-- Otherwise wimply display the string -->
+        <ng-container *ngIf="!isTemplate">
+            <p>{{ description }}</p>
+        </ng-container>
+    </div>
+
+    <ul class="marquee-wizard-steps">
+
+        <li class="marquee-wizard-step" *ngFor="let step of steps" (click)="gotoStep(step)" [class.active]="step.active" [class.visited]="step.visited" [class.invalid]="!step.valid">
+            <i class="marquee-wizard-step-icon" [ngClass]="step.icon"></i>
+            <span class="marquee-wizard-step-title">{{ step.header }}</span>
+            <span class="marquee-wizard-step-status hpe-icon hpe-checkmark" *ngIf="step.completed"></span>
+        </li>
+
+    </ul>
+</div>
+
+<div class="marquee-wizard-content-panel">
+    <div class="marquee-wizard-content">
+        <ng-content></ng-content>
+    </div>
+
+    <div class="marquee-wizard-footer">
```

Comment:
I think if you use `.btn-container` here, it give correct flex layout and spacing between buttons. Also I think `.modal-footer` is similar.